app/models/escort.rb

```diff
@@ -29,4 +29,16 @@ def offences
   def risks
     super || build_risks
   end
+
+  def with_future_move?
+    true if move.date && move.date > Date.current
```

Comment:
There's no need for the `true if` - this method can just return the boolean outcome of the logical &&